results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,12 @@
+The Red Hat JBoss Developer Studio Platform Installer scans for the products listed below using your PATH environment variable to determine if they're already installed. If not found, or an older version is installed, the following supported versions will then be installed for you by the installer:
```

Comment:
It's not quite true yet. We do re-use vagrant and virtualbox if they already installed. The rest will be always installed. We are going to allow to re-use all of the components in the next release.